src/main/java/com/epam/test_generator/services/UserService.java

```diff
@@ -67,6 +68,7 @@ public void saveUser(User user) {
     /**
      * Creates in database user with admin role if none exists
      */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Add info about propagation level to javadoc

Reply:
what should I add about propagation?

Reply:
Why REQUIRES_NEW exactly.